sources/addr.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "addr",
 	"name": "addr",
 	"prefix": "addr",
-    	"key":"id",
+	"key": "id",
 	"url": "",
-	"license": "CC0",
-	"description": ""
-}
+	"license": "N/A",
+	"license_type": "CC0",
```

Comment:
I searched a venue repo and the `whosonfirst-data` repo and didn't see any `src:addr` properties, so I don't think any info about that field needs to be added. I've updated this property to `"N/A"`, though, in the most recent commit..